vault/logical_system.go

```diff
@@ -3003,6 +3003,45 @@ func (*SystemBackend) handlePoliciesPasswordGenerate(ctx context.Context, req *l
 	return resp, nil
 }
 
+// handlePoliciesDetailedAclList handles the listing of ACL policies with detailed information
+func (b *SystemBackend) handlePoliciesDetailedAclList() framework.OperationFunc {
+	return func(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+		// Get policies list
+		policies, err := b.Core.policyStore.ListPolicies(ctx, PolicyTypeACL)
+		if err != nil {
+			return nil, err
+		}
+
+		// Add root policy if in root namespace
+		ns, err := namespace.FromContext(ctx)
+		if err != nil {
+			return nil, err
+		}
+		if ns.ID == namespace.RootNamespaceID {
```

Comment:
@voigt With the changes to seal namespace, will we be able to create a namespace-bound root token through the use of recovery keys? :thinking: